templates/snap-details.html

```diff
@@ -3,42 +3,48 @@
 {% block title %}{{ name }} snap details{% endblock %}
 
 {% block content %}
-  <div class="row">
-    <div class="col-8">
-      <h1>{{ name }}</h1>
+  <div class="p-strip">
+    <div class="row">
+      <div class="col-2">
+        {% if icon_url %}
+          <img src="{{ icon_url }}" alt="{{ name }} snap" />
+        {% else %}
+          <img src="https://assets.ubuntu.com/v1/610689b1-default-package-icon.svg" alt="" />
```

Comment:
Not sure what our accessibility guidelines are for such situations.

Looking at https://www.w3.org/WAI/tutorials/images/ I would say that snap icon falls into 'Decorative images' category (because the image itself doesn't really give any more info to the user), so we should probably have empty `alt` in both cases (when there is icon and when there is not).
Because what value there is to someone who doesn't see the icon to hear that it is there?